dsb-gui/build.gradle

```diff
@@ -73,7 +73,7 @@ classes.dependsOn genConfigProperties
 
 ospackage {
     packageName = 'eonbrowser'
-    version = '5.0.1'
+    version = '5.0.2'
```

Comment:
We should be able to pull this version from the project.  That way we won't need to update it every time.

Reply:
:+1: Let me get that fixed real quick!